### ISSUE TITLE
`FAILED to handle event: TypeError: Cannot read properties of undefined` when opening HTML file

### ISSUE THREAD

**Issue by aaronbushnell** [Stillat/vscode-antlers-language-server#88]
**Describe the bug**
When I open an HTML file and have the VS Code "Output" shown with "Window" selected from the dropdown I get the following error:

```
2024-06-05 20:00:01.260 [error] [ExtensionListenerError] Extension 'stillat-llc.vscode-antlers' FAILED to handle event: TypeError: Cannot read properties of undefined (reading 'document'): Error: [ExtensionListenerError] Extension 'stillat-llc.vscode-antlers' FAILED to handle event: TypeError: Cannot read properties of undefined (reading 'document')
    at n.value (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:163:20749)
    at l.y (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:82:664)
    at l.z (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:82:734)
    at l.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:82:950)
    at n.value (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:155:31896)
    at l.y (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:82:664)
    at l.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:82:881)
    at m.acceptDocumentsAndEditorsDelta (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:155:30786)
    at m.$acceptDocumentsAndEditorsDelta (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:155:29168)
    at E.S (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:150:5970)
    at E.Q (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:150:5736)
    at E.M (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:150:4762)
    at E.L (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:150:3589)
    at n.value (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:150:2282)
    at l.y (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:82:664)
    at l.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:82:881)
    at s.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:107:14502)
    at n.value (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:176:8481)
    at l.y (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:82:664)
    at l.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:82:881)
    at s.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:107:14502)
    at MessagePortMain.<anonymous> (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:176:6631)
    at MessagePortMain.emit (node:events:514:28)
    at Object.MessagePortMain._internalPort.emit (node:electron/js2c/utility_init:2:2285)
```

**Impacted Products**
Which Antlers Toolbox products does this bug apply to?

- [x] Antlers Toolbox for VS Code
- [ ] Formatter CLI
- [ ] Antlers Prettier Plugin
- [ ] Antlers Syntax Highlighting (external systems - not VS Code)

**Versions and Other Plugins/Extensions**
- stillat-llc.vscode-antlers 2.6.17
- VS Code 1.90.0 

**To Reproduce**
- Install Antlers Toolbox
- Open up the Output window and make sure "Window" is selected
- Open HTML files

**Comment by JohnathonKoster** [comment]
Thanks for the report! A fix for this is now available in `2.6.18`

**Comment by aaronbushnell** [comment]
Wow, thanks so much for that quick fix, @JohnathonKoster!
